tests/models/test_p12.py

```diff
@@ -0,0 +1,73 @@
+from pathlib import Path
+
+from codemagic.models import Certificate
+from codemagic.models import PrivateKey
+
+certificate_bytes = \
```

Comment:
`certificate_bytes` is initiated indicating that the type of value is `bytes`, but it is `str` instead.
```suggestion
certificate_pem = \
```